tasks-by-users/nehalskdev/02 Poject/Thanks.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Thanks, Appreciation">
```

Comment:
what is the use of this ?

Reply:
it came with shift+! i forgot to remove. now i have removed it as per your comment